[2025-02-25T13:28:11.922+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweather_data_lake.initialize_buckets scheduled__2025-02-25T13:27:00+00:00 [queued]>
[2025-02-25T13:28:11.925+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweather_data_lake.initialize_buckets scheduled__2025-02-25T13:27:00+00:00 [queued]>
[2025-02-25T13:28:11.925+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2025-02-25T13:28:11.930+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): initialize_buckets> on 2025-02-25 13:27:00+00:00
[2025-02-25T13:28:11.933+0000] {standard_task_runner.py:57} INFO - Started process 1474 to run task
[2025-02-25T13:28:11.934+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'openweather_data_lake', 'initialize_buckets', 'scheduled__2025-02-25T13:27:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/openweather_data_lake.py', '--cfg-path', '/tmp/tmpliz8_vms']
[2025-02-25T13:28:11.936+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask initialize_buckets
[2025-02-25T13:28:11.943+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-25T13:28:11.957+0000] {task_command.py:415} INFO - Running <TaskInstance: openweather_data_lake.initialize_buckets scheduled__2025-02-25T13:27:00+00:00 [running]> on host 90c8125782a4
[2025-02-25T13:28:11.990+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweather_data_lake' AIRFLOW_CTX_TASK_ID='initialize_buckets' AIRFLOW_CTX_EXECUTION_DATE='2025-02-25T13:27:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-25T13:27:00+00:00'
[2025-02-25T13:28:12.212+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'raw' existe déjà.
[2025-02-25T13:28:12.212+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'staging' existe déjà.
[2025-02-25T13:28:12.212+0000] {logging_mixin.py:151} INFO - [INFO] Bucket 'curated' existe déjà.
[2025-02-25T13:28:12.213+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-25T13:28:12.218+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=openweather_data_lake, task_id=initialize_buckets, execution_date=20250225T132700, start_date=20250225T132811, end_date=20250225T132812
[2025-02-25T13:28:12.242+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-25T13:28:12.253+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
